android/anc/src/main/java/org/smartregister/fhircore/anc/ui/login/AncLoginService.kt

```diff
@@ -19,16 +19,48 @@ package org.smartregister.fhircore.anc.ui.login
 import android.content.Intent
 import javax.inject.Inject
 import org.smartregister.fhircore.anc.ui.family.register.FamilyRegisterActivity
+import org.smartregister.fhircore.anc.ui.pin.PinLoginActivity
+import org.smartregister.fhircore.anc.ui.pin.PinSetupActivity
 import org.smartregister.fhircore.engine.ui.login.LoginActivity
 import org.smartregister.fhircore.engine.ui.login.LoginService
 
 class AncLoginService @Inject constructor() : LoginService {
 
   override lateinit var loginActivity: LoginActivity
 
-  override fun navigateToHome() {
+  /**
+   * Navigate to app home page as this fun is implemented if multiple modules,
+   * @param canSetPin : boolean to check whether app can navigate Home or PinSetup page
+   */
+  override fun navigateToHome(canSetPin: Boolean) {
```

Comment:
@pld : not sure why, as its already implemented, I need to adjust by overriding and bypassing Home to PinSetup Screen, its app is configured via eCBIS.

Reply:
The plan w/the next round of eCBIS is to use Quest, so we'd want any PIN logic to be somewhere generic where Quest can use, and make sure it works in Quest, instead of having anything tied to ANC